src/views/queries/responseCo2scope3.tsx

```diff
@@ -273,7 +273,8 @@ export default class Scope3CarbonConsumptionResponseTemplates {
         <td>{Html.escapeHtml(props.company_name)}</td>
         <td>
           <input
-            name={`${props.id}[]`}
+            id={props.id}
+            name={`connections`}
```

Comment:
what is the the  `name={`connections`}` for?

Reply:
example might be the best option here, basically it groups html inputs into an array so we can submit as per below example:

```
{
  "companyId": "7c36fd85-7f4e-4330-ad3c-901b51e43712",
  "partialQuery": "on",
  "connections": [
    "7c36fd85-7f4e-4330-ad3c-901b51e43713", // uuid of connection
    "a", // product_id
    "1", // quantity
    "7c36fd85-7f4e-4330-ad3c-901b51e43714",
    "b",
    "2",
    "7c36fd85-7f4e-4330-ad3c-901b51e43715",
    "c",
    "3"
  ],
  "action": "success"
}

/// another one
{
  "companyId": "7c36fd85-7f4e-4330-ad3c-901b51e43712",
  "connections": [
    "7c36fd85-7f4e-4330-ad3c-901b51e43713",
    "product_a",
    "10",
    "7c36fd85-7f4e-4330-ad3c-901b51e43714",
    "product_b",
    "10"
  ],
  ...
  "action": "success"
}
```

Reply:
if it groups inputs into an array ... you do not need a different identifier/name for each row then?

Reply:
So this has changed, just doing some final sanity checks and will update, but in the nutshell it will look like the below also there is an aggregate function e.g. will put into an array of objects.

```js
  "companyId": "7c36fd85-7f4e-4330-ad3c-901b51e43712",
  "7c36fd85-7f4e-4330-ad3c-901b51e43713": ['7c36fd85-7f4e-4330-ad3c-901b51e43713', 'product-x', 10],
  "7c36fd85-7f4e-4330-ad3c-901b51e43711": ['7c36fd85-7f4e-4330-ad3c-901b51e43711', 'product-a', 110],
  "7c36fd85-7f4e-4330-ad3c-901b51e43712": ['7c36fd85-7f4e-4330-ad3c-901b51e43712', 'product-b', 100],
    ...
  "action": "success"
```